File: backend/app/osm/highway_include.py
"""本番・デバッグ UI で共通に使う highway=* 値（道路ネットワークに含める種別）。"""

# frontend/src/lib/highwayInclude.ts の INCLUDED_HIGHWAY_TYPES と同期して保つこと。
INCLUDED_HIGHWAY_TYPES: tuple[str, ...] = (
    "trunk",
    "primary",
    "secondary",
    "tertiary",
    "unclassified",
    "residential",
    "service",
)

File: frontend/src/lib/highwayInclude.ts
/** 道路ネットワークに含める highway=* 値。backend/app/osm/highway_include.py と同期して保つ。 */

export const INCLUDED_HIGHWAY_TYPES = [
  'trunk',
  'primary',
  'secondary',
  'tertiary',
  'unclassified',
  'residential',
  'service',
] as const

export type IncludedHighwayType = (typeof INCLUDED_HIGHWAY_TYPES)[number]
